scripts/sbc-add-broker.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+set -euo pipefail
+IFS=$'\n\t'
+
+DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null && pwd )"
+source ${DIR}/helper/functions.sh
+source ${DIR}/env.sh
+
+#-------------------------------------------------------------------------------
+
+echo "Starting broker kafka3, then tailing SBC logging output.  Type CTRL-C to stop tailing logs."
+docker-compose up -d kafka3
+docker-compose logs -f --tail=1000 | grep -E "(databalancer|cruisecontrol)" | grep -v "confluent.balancer.class"
```

Comment:
I'm inclined to not have a script that just keeps running with a log tail.  For consideration: maybe it's better to do something like (pseudo code):

1. bring up kafka3
2. set a wait time and run `retry` to look for a specific message in the log
3. exit when log message is seen or max wait time is reached

Reply:
Agreed, the foreground log tail isn't great.

The main motivation for the script in the first place, is that we don't have first-class support for demo lifecycle outside of `start.sh` and `stop.sh`, so the script sources `env.sh` to avoid needing to do that manually.  Then there's some monitoring support in Control Center but the logs provide a lot of details.  The SBC process can be long-running, so I'm not sure it fits log-monitor retry/timeout perfectly ... feels more like something that the curious can monitor, with perhaps some better log filtering.

I'm somewhat inclined to drop the script altogether and do this via documented command, like:

- `(source scripts/env.sh;docker-compose up -d kafka3)`
- `docker-compose logs -f | grep "com.linkedin.kafka.cruisecontrol.executor.Executor"`

... which can be used to optionally follow SBC progress.

The documentation will describe the required `kafka-topics` commands to run to see the changes made by SBC once Control Center and logs confirm completion.

WDYT?

Reply:
Please see https://github.com/confluentinc/cp-demo/pull/353#discussion_r594280310 for comment on above.